hera_qm/version.py

```diff
@@ -61,6 +80,7 @@ def construct_version_info():
                             + '.  Git hash: ' + git_hash
                             + '.  Git branch: ' + git_branch
                             + '.  Git description: ' + git_description + '.')
+hera_qm_version_str = str(hera_qm_version_str)
```

Comment:
You might be interested in this nicely formatted history string function I produced for hera_cal: https://github.com/HERA-Team/hera_cal/blob/df1009b80773cb8c304accc92f5e5c8c0d5b482b/hera_cal/version.py#L84

Here's an example: 
```
------------
This file was produced by the function post_redcal_abscal_run() in abscal.py using:
    git_branch: master
    git_description: v2.0-603-gdf1009b
    git_hash: df1009b80773cb8c304accc92f5e5c8c0d5b482b
    git_origin: https://github.com/HERA-Team/hera_cal.git
    version: 2.0
------------
```

Reply:
This is a good idea, but I'm going to punt on it for this PR. I only edited the version file because things started failing on travis. Otherwise this is pretty unrelated. I've made an issue so we don't lose track of it.